plugins/gateway/kong/plugin.go

```diff
@@ -113,11 +113,11 @@ func shouldTrace(kong *pdk.PDK) (bool, error) {
 	if err != nil {
 		return false, fmt.Errorf("failed to get routed service: %v", err)
 	}
-	host, _ := parseHost(routedService.Host)
-	if traceSamplingClient.ShouldTrace(host) {
+	host, port, _ := parseKongHost(routedService.Host)
+	if traceSamplingClient.ShouldTrace(host, port) {
 		return true, nil
 	}
-	_ = kong.Log.Info("Ignoring host: %v", host)
+	_ = kong.Log.Info("Ignoring host: %v:%v", host, port)
```

Comment:
Are we showing this log, with INFO level, for each and every ignored traces ?

Reply:
Yes